### DISCUSSION TITLE
Could not start fs_event watcher in macos error

### DISCUSSION THREAD

**Discussion by nishu-murmu** [nvim-tree/nvim-tree.lua#1585] [Q&A] [answered]
![Screenshot 2022-09-07 at 10 58 23 AM](https://user-images.githubusercontent.com/71123337/188795981-863c19d3-6fd4-4e18-818c-6a699bdc670d.png)
How to solve this error?

**Comment by alex-courtis** [comment, marked as answer]
It looks like your non-login shell prints information. That is a Bad Idea. See "STARTUP/SHUTDOWN FILES" in `man zsh`.

`.zshenv` (non-login shell) should be minimal, setting only environment variables and non-interactive functions.

Everything else should go `.zshrc` (interactive shell) and `.zprofile` (login shell): aliases, interactive functions, prompts, __git_ps1, zle, tmux etc. etc.

Test:
```sh
zsh -c "echo nothingbutthis"
nothingbutthis
```

**Reply by nishu-murmu** [reply]
Thanks anyway,
I solved this problem by removing .git folder from my HOME directory
